freebsd/FreeBSDProcessList.c

```diff
@@ -82,6 +84,9 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* dynamicMeters, H
    len = 4; sysctlnametomib("vm.stats.vm.v_cache_count", MIB_vm_stats_vm_v_cache_count, &len);
    len = 4; sysctlnametomib("vm.stats.vm.v_inactive_count", MIB_vm_stats_vm_v_inactive_count, &len);
    len = 4; sysctlnametomib("vm.stats.vm.v_free_count", MIB_vm_stats_vm_v_free_count, &len);
+   MIB_vm_vmtotal[0] = CTL_VM;
+   MIB_vm_vmtotal[1] = VM_TOTAL;
+
```

Comment:
Note where the blank line should go.
```suggestion

   MIB_vm_vmtotal[0] = CTL_VM;
   MIB_vm_vmtotal[1] = VM_TOTAL;
```

That aside: Why not use the sysctlnametomib for lookup here too? If this was really a constant better push these values as the initializer and make the actual variable const.

Reply:
I replaced it with a `sysctlnametomib` lookup.